app/templates/views/guidance/letter-specification.html

```diff
@@ -0,0 +1,47 @@
+{% extends "content_template.html" %}
+{% from "components/service-link.html" import service_link %}
+
+{% block per_page_title %}
+  Letter specification
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Letter specification (version 2.4)</h1>
+
+  <p class="govuk-body">Page size and layout: A4 portrait (210 × 297 mm)</p>
+  <p class="govuk-body">Maximum file size: 2 MB</p>
+  
+  <p class="govuk-body">Your letter must 10 pages or less (5 double-sided sheets of paper).</p>
+  <p class="govuk-body">Content must be inside the printable area.</p>
+  
+  <h2 class="heading-medium">Page 1 of 10</h2>
+  
+  <p class="govuk-body">Top margin 5.00mm</p>
+  <p class="govuk-body">Left margin 15.00mm</p>
+  <p class="govuk-body">Right margin 15.00mm</p>
+  <p class="govuk-body">Bottom margin 5.00mm</p>
+
+  <h3 class="heading-small">Logo block</h3>
+  
+  <p class="govuk-body">Position: 5mm from top edge, 15mm from left edge</p>
+  <p class="govuk-body">Size: 200mm wide by 25mm high</p>
+  
+  <h3 class="heading-small">Address block</h3>
+  
+  <p class="govuk-body">Position: 24.6mm from top edge, 39.5mm from left edge</p>
+  <p class="govuk-body">Size: 95.4mm wide by 26.8mm high</p>
+  
+  <h3 class="heading-small">Letter contact block</h3>
```

Comment:
```suggestion
  <h3 class="heading-small">Contact block</h3>
```
Doesn't seem necessary to repeat `letter` here

Reply:
`Letter contact block` is what we call it in the UI. I'm sure we talked about changing it at some point, because we saw a few users struggling to understand what it was in usability testing.

Reply:
I think use _Letter contact block_ for consistency then. If we change it to something better we should do that everywhere (which is out of the scope of these accessibility improvements).